package.json

```diff
@@ -68,20 +68,30 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "rimraf": "^3.0.2",
+    "sb": "7.0.0-alpha.29",
     "svelte": "^3.50.0",
     "svelte-jester": "^2.3.2",
     "svelte-loader": "^3.1.2",
-    "typescript": "^3.9.7"
+    "typescript": "^3.9.7",
+    "vite": "^3.1.4"
   },
   "peerDependencies": {
     "@storybook/svelte": ">=6.4.20",
     "@storybook/theming": ">=6.4.20",
```

Comment:
Is this correct?  Shouldn't these be 7.0?

Reply:
You're right. Fixing in another PR